src/toil/cwl/cwltoil.py

```diff
@@ -1033,7 +1043,9 @@ def __str__(self) -> str:
 
 class ToilCommandLineTool(ToilTool, cwltool.command_line_tool.CommandLineTool):
     """Subclass the cwltool command line tool to provide the custom ToilPathMapper."""
-
+    def __init__(self, toolpath_object: 'CommentedMap', loadingContext: 'cwltool.context.LoadingContext'):
```

Comment:
```suggestion
    def __init__(self, toolpath_object: CommentedMap, loadingContext: cwltool.context.LoadingContext):
```
These types aren't hidden behind a `if TYPE_CHECKING:` block, so you can use them directly.

Reply:
I removed this addition since it's no longer necessary when passing `--relax-path-checks` to the runner